src/test/java/com/amazon/ion/impl/bin/IonRawBinaryWriterTest_1_1.kt

```diff
@@ -33,8 +35,16 @@ class IonRawBinaryWriterTest_1_1 {
      *                 you can use `|` to add comments.
      */
     private inline fun assertWriterOutputEquals(hexBytes: String, autoClose: Boolean = true, block: IonRawBinaryWriter_1_1.() -> Unit) {
-        val hexBytes = hexBytes.replace(Regex("\\s+(\\|.*\\n)?\\s+"), " ").trim().uppercase()
-        assertEquals(hexBytes, writeAsHexString(autoClose, block))
+        val commentRegex = Regex("\\|.*$")
+        val excessWhitespaceRegex = Regex("\\s+")
+        val cleanedHexBytes: String = hexBytes.lines()
+            .map { it.replace(commentRegex, "").trim() }
+            .filter { it.isNotBlank() }
+            .joinToString(" ")
+            .replace(excessWhitespaceRegex, " ")
+            .uppercase()
+            .trim()
```

Comment:
🗺️ I had to fix this up a bit so that it could correctly handle lines that have comments but no hex pairs. E.g.
```
C2          | {
03 5E       |    $1: true
            | }
```